src/conduit/mixins.cljs

```diff
@@ -24,9 +24,19 @@
 (defn form [{:keys [fields validators on-submit]}]
   (let [data-init (->> fields keys (reduce #(assoc %1 %2 "") {}))
         errors-init (->> fields keys (reduce #(assoc %1 %2 nil) {}))
-        fields-init fields
         data (atom data-init)
         errors (atom errors-init)
+        fields-init (->> fields
```

Comment:
This code looks complex. Also I think it should be gone since `:container` and `:events` config fields is a bit overgeneralization for a single component like `InputField`.

Reply:
Since form data and form errors are not exposed outside of the form mixin for altering, there is no way to set a new form property for tags. So I come up with this hacky solution by exposing form data form errors atoms to `:container` and `:events` for altering, which should make very minimal impact on the form state while still keeping it general enough for other situations. I know it doesn't sound like a good idea, but I think it serves the purpose for now.